internal/apiserver/route_post_verifiers_resolve.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"net/http"
+
+	"github.com/hyperledger/firefly-common/pkg/ffapi"
+	"github.com/hyperledger/firefly/internal/coremsgs"
+	"github.com/hyperledger/firefly/pkg/core"
+)
+
+var postVerifiersResolve = &ffapi.Route{
+	Name:            "postVerifiersResolve",
+	Path:            "verifiers/resolve",
+	Method:          http.MethodPost,
+	PathParams:      nil,
+	QueryParams:     nil,
+	Description:     coremsgs.APIEndpointsPostVerifiersResolve,
+	JSONInputValue:  func() interface{} { return &core.VerifierRef{} },
+	JSONOutputValue: func() interface{} { return &core.VerifierRef{} },
+	JSONOutputCodes: []int{http.StatusOK},
+	Extensions: &coreExtensions{
+		CoreJSONHandler: func(r *ffapi.APIRequest, cr *coreRequest) (output interface{}, err error) {
+			return cr.or.ResolveSigningKey(cr.ctx, r.Input.(*core.VerifierRef))
```

Comment:
I am calling `ResolveSigningKey()` from `Orchestrator` to handle the request. Should I move `ResolveSigningKey()` to the blockchain plugin instead?